build/cmake/DefinePlatformSpecifc.cmake

```diff
@@ -97,21 +97,14 @@ elseif(WITH_STDTHREADS)
 endif()
 
 # C++ Language Level
-set(CXX_LANGUAGE_LEVEL "C++${CMAKE_CXX_STANDARD}")
-if (CMAKE_CXX_STANDARD_REQUIRED)
-  string(CONCAT CXX_LANGUAGE_LEVEL "${CXX_LANGUAGE_LEVEL} [compiler must support it]")
-else()
-  string(CONCAT CXX_LANGUAGE_LEVEL "${CXX_LANGUAGE_LEVEL} [fallback to earlier if compiler does not support it]")
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
By setting the standard outright, one cannot configure it on the build line to C++14, C++17, or C++2a.  Can we set it to 11 if it isn't already set?

Reply:
@jeking3  fixed.